python/ml4ir/base/model/architectures/dnn.py

```diff
@@ -21,7 +23,12 @@ class DNNLayer:
 class DNN:
     def __init__(self, model_config: dict, feature_config: FeatureConfig, file_io):
         self.file_io: FileIO = file_io
+        self.model_config = model_config
+        self.feature_config = feature_config
         self.layer_ops: List = self.define_architecture(model_config, feature_config)
+        if 'positional_bias_handler' in self.model_config and self.model_config['positional_bias_handler'][
```

Comment:
@mohazahran We discussed to move the positional bias under `layers` and thereby under `define_architecture()`, no? That would be clean I think as it is in essence a layer of the DNN.

Reply:
The added advantage is that it will automatically get added in `get_architecture_op`

Reply:
You can add `positional_bias` to `DNNLayer` keys as well.

Reply:
Yeah but we discussed that doing so would put responsibility on the user to specify that the positional bias layer must be the last layer. I don't think it's a good thing to do.